sycl/include/sycl/stl_wrappers/corecrt.h

```diff
@@ -36,6 +36,14 @@ extern "C" inline void __cdecl _invalid_parameter(wchar_t const *,
   // Do nothing when called in device code
 }
 
+extern "C" __declspec(noreturn) void __cdecl
```

Comment:
Can you update the comment at line 22?

Reply:
I've reworked the entire comment. Hopefully, it is succinct and clear now.